### ISSUE TITLE
If your having issues installing due to an install error read this!

### ISSUE THREAD

**Issue by PS2ClassicsVault** [ACCESS-DENIIED/Decky-Loader-For-Windows#2]
I had this same issue where it would error out saying something about the project is not a Python blah blah blah or something like that, for you to get it to install and run first run the exe first let it install the necessary dependencies first

Then when it errors out in the same folder you will  see a file called decky_builder.py and run that. It will open Python via a command prompt and will now install decky_loader properly once it completes you will see two steam icons, however before you reboot just make sure big picture mode is set to be on by default then reboot your steam deck.

Now make sure your using steam controller configuration to open the QAM to access decky loader.

Beyond that this should help until the developer fixes this issue!

Also to @ACCESS-DENIIED could u make it where it auto logs into an account in every boot instead of asking me to choose the language and account manually every time

**Comment by PS2ClassicsVault** [comment]
> I appreciate your powers of deduction. Yes, the python script runs perfectly! But the intended experience is to basically take a brand new pc / handheld and have 1 installer for all dependencies required for decky loader. So for users such as yourself and I, the python script works great! Hoping to squash the bugs out by the end of the week.
> 
> And I actually wasn't aware that this happened. It doesn't occur on my machine so I'll run some tests to try to reproduce the steps and go from there. Thanks!

Your welcome and thanks

**Comment by ACCESS-DENIIED** [comment]
Hi. Can you let me know if you're issue has been resolved in the latest version?

**Comment by PS2ClassicsVault** [comment]
> Hi. Can you let me know if you're issue has been resolved in the latest version?

Yes using the latest pre release you released and I love it and it fixed all the issues I was having thank you!
